src/Vijay/Week4/MajorityElement.java

```diff
@@ -0,0 +1,62 @@
+package Vijay.Week4;
+
+import java.util.Arrays;
+
+public class MajorityElement {
+    public int majorityElement(int[] nums) {
+        // int globalCounter = 1;
+        // int localCounter = 1;
+        // int majorityElement = nums[0];
+        // for(int i = 0; i < nums.length; i++) {
+        //     majorityElement = nums[i];
+        //     for(int j = i + 1; j < nums.length; j++) {
+        //         if(nums[j] == majorityElement) {
+        //             localCounter++;
+        //         }
+        //     }
+        //     if(localCounter > globalCounter) {
+        //         globalCounter = localCounter;
+        //     }
+        //     localCounter = 1;
+        // }
+        // return majorityElement;
+
+// Tried to shorten the runtime by sorting the array so that loop would not run for repetitive numbers.
+// So the below logic is based on the fact that you are sorting the array otherwise
+// it will not pass for a case like: [2,2,1,1,1,2,2]
+        Arrays.sort(nums); // comment this and the below logic will fail!
```

Comment:
I think both commented code and the below code are same.

Reply:
No, not exactly the same, but are only different in sorting and without sorting.